incubator/etcd/templates/statefulset.yaml

```diff
@@ -34,141 +34,216 @@ spec:
             memory: "{{ .Values.Memory }}"
         env:
         - name: INITIAL_CLUSTER_SIZE
-          value: {{ default 3 .Values.Replicas | quote }}
+          value: "1"
```

Comment:
as above, why explicit value here?

Reply:
The mechanism changed, as described in the README.
A cluster is initially 1 node. so if you specify 3 replicas in values.yaml, the first replica forms a 1 node cluster and the remaining 2 are added to the cluster. This allows the cluster to come down to 1 node (for whatever reason) without breaking.